pkg/test/framework/components/echo/native/workload.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 const (
-	envoyLogLovel = envoy.LogLevelWarning
+	envoyLogLovel = envoy.LogLevelTrace
```

Comment:
Do we want this or was it for debugging?

Reply:
Lovel => Level